src/main/scala/rps/WebServer.scala

```diff
@@ -0,0 +1,44 @@
+package rps
+
+import rps.dto._
+import rps.model._
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives._
+import akka.stream.ActorMaterializer
+import scala.io.StdIn
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
+import io.buildo.enumero.circe._
+import io.circe._, io.circe.generic.auto._
+import io.circe.syntax._
+
+object WebServer {
+  def main(args: Array[String]) {
+    implicit val system = ActorSystem("my-system")
+    implicit val materializer = ActorMaterializer()
+
+    implicit val executionContext = system.dispatcher
+    val route =
+      
+      pathPrefix("rps") {
+        path("play") {
+          post {
+             entity(as[Request]) { request =>
+              val round = Game.play(request.userMove)
+              val response = new Response(round)
+              complete(HttpEntity(ContentTypes.`application/json`, response.asJson.noSpaces))
```

Comment:
with the right imports in scope (and its looks like they are), you can simply do `complete(response)`

That's the purposes of `de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport`: it provides the bridge between circe and akka-http marshallers.

Reply:
A small difference though, following this suggestion the answer came out in this way
    "{\"userMove\":\"Paper\",\"computerMove\":\"Scissors\",\"result\":\"Lose\"}"
Instead of 
    {"userMove":"Paper","computerMove":"Scissors","result":"Lose"}

I will also keep it this way?

Reply:
That looks simply encoded. If the client responds correctly, then it's ok.